backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/CStubsManager.kt

```diff
@@ -18,13 +20,24 @@ class CStubsManager {
     fun compile(clang: ClangArgs, messageCollector: MessageCollector, verbose: Boolean): File? {
         if (stubs.isEmpty()) return null
 
-        val cSource = createTempFile("cstubs", ".c").deleteOnExit()
+        val compilerOptions = mutableListOf<String>()
```

Comment:
Maybe add TODO about better file extension computation?

Reply:
This is exactly what I've meant in the TODO below.